Adds a new datagen input adapter for tables. (#2107)
feldera/feldera: docs/tutorials/basics/part3.md

# Part 3: Input and Output Connectors

> This is the key part of the tutorial where we will

Reviewer: ```suggestion
In this part of the tutorial where we will
```
